worlds/lingo/test/__init__.py

```diff
@@ -0,0 +1,43 @@
+from typing import ClassVar
+
+from argparse import Namespace
+from BaseClasses import MultiWorld
+from test.TestBase import WorldTestBase
+from test.general import gen_steps
+from .. import LingoWorld
+from ... import AutoWorld
+from ...AutoWorld import call_all
+
+
+class LingoTestBase(WorldTestBase):
+    game = "Lingo"
+    world: LingoWorld
+    player: ClassVar[int] = 1
+
+    def world_setup(self, *args, **kwargs):
+        # Most of this is copied from the superclass, because we want to run some code in the middle of generation.
```

Comment:
why not just call super? if you explicitly need to know you're generating the world from a unit test you can set an env variable globally from this module and check for that.

Reply:
yeah i still don't like this. zillion has to do similar so would recommend using a similar approach as what it does here. `/zillion/__init__.py` L149